common/production_methods/11_private_infrastructure.txt

```diff
@@ -7,6 +7,7 @@
 			goods_input_coal_add = 2 # 60
 			goods_output_transportation_add = 20 # 600
 		}
+		#(20*30)-((5*60)+(2*30)) = 240*52= 12,480
```

Comment:
I'm sorry, what?

Reply:
profitability calculation I made in a note regarding railways. mostly for myself.